rainbow-cake-core/src/main/java/co/zsmb/rainbowcake/config/RainbowCakeConfigurator.kt

```diff
@@ -37,11 +55,18 @@ public interface RainbowCakeConfigurator {
      * Default value: true.
      */
     public var consumeExecuteExceptions: Boolean
-
 }
 
 /**
  * Whether the application is running in production.
  */
 internal val RainbowCakeConfigurator.isProd
     get() = !isDebug
+
+/**
+ * Whether the application is not allowed to send log.
+ *
+ * Note: [isProd] is used for now to ensure retro-compatibility for release builds !
+ */
+internal val RainbowCakeConfigurator.isNotLoggable
+    get() = !isLoggable || isProd
```

Comment:
As described in kdoc, this method is thought to have no impact in app using `isDebug` to not logging in production